### PR TITLE
Add Edge Identity README

### PR DESCRIPTION
**Pull request by kevinlind**
Adds the install instructions for the Adobe Experience Platform Edge Identity Extension. Includes instructions for installing Edge Identity alone and installing Edge Identity with Identity.
Other Edge Identity documentation, such as API usage, will be in another PR.

Internal ticket: AMSDK-11053

### CODE REVIEW

File: using-mobile-extensions/adobe-edge-identity/README.md [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+    MobileCore.registerExtensions([Identity.self, Edge.self], {
+    MobileCore.configureWith(appId: "yourLaunchEnvironmentID")
+  })
+  ...
+}
+```
+
+### Objective-C
+
+```text
```
Comment (PravinPK) [inline using-mobile-extensions/adobe-edge-identity/README.md:117]: can we try the `objectivec` tag not sure if it makes difference. But I presume its better than text

File: using-mobile-extensions/adobe-edge-identity/README.md [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+// AppDelegate.swift
+import AEPCore
+import AEPIdentity
+import AEPEdge
+import AEPEdgeIdentity
+```
+
+### Objective-C
+
+```text
```
Comment (PravinPK) [inline using-mobile-extensions/adobe-edge-identity/README.md:182]: same here objectivec tag

**Review by PravinPK — COMMENTED** [review]
Looks good to me

**Review by PravinPK — APPROVED** [review]

File: using-mobile-extensions/adobe-edge-identity/README.md [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+2. On the **Catalog** tab, locate or search for the **Edge Identity** extension, and click **Install**.
+3. There are no configuration settings for **Edge Identity**.
+4. Click **Save**.
+5. Follow the publishing process to update SDK configuration.
+
+## Add the AEP Edge Identity extension to your app
+
+### Download and import the Edge Identity extension
+{% hint style="info" %}
+The following instructions are for configuring an application with Edge Identity. If an application will include both Identity and Edge Identity extensions, follow the instructions [below](#download-and-import-the-identity-and-edge-identity-extensions).
```
Comment (emdobrin) [inline using-mobile-extensions/adobe-edge-identity/README.md:16]: ... are for configuring an application using Adobe Experience Platform Edge mobile extensions. If an application includes both Edge Network and Adobe Solution extensions, both the Edge Identity and Identity extensions are required, find more details in the Frequently Asked Questions page <link>.

File: using-mobile-extensions/adobe-edge-identity/README.md [hunk 4]
```diff
@@ -0,0 +1,258 @@
...
+
+      try {
+        Edge.registerExtension();
+        Identity.registerExtension();
+        MobileCore.start(new AdobeCallback() {
+          @Override
+          public void call(final Object o) {
+            // processing after start
+          }});
+      } catch (Exception e) {
```
Comment (emdobrin) [inline using-mobile-extensions/adobe-edge-identity/README.md:94]: there is no exception thrown here.

File: using-mobile-extensions/adobe-edge-identity/README.md [hunk 5]
```diff
@@ -0,0 +1,258 @@
...
+    ...
+  }];
+  [AEPMobileCore configureWithAppId: @"yourLaunchEnvironmentID"];
+  ...
+}
+```
+{% endtab %}
+{% endtabs %}
+
+### Download and import the Identity and Edge Identity extensions
```
Comment (emdobrin) [inline using-mobile-extensions/adobe-edge-identity/README.md:130]: Can we move this in an FAQ page and formulate it like this:
Q: I am using AEP Edge and Adobe Solutions extensions, which Identity Extension should I install and register?
A: Both. Mention the extension names and provide sample with registration code for both.

File: using-mobile-extensions/adobe-edge-identity/README.md (hunk 4 above)
Comment (nporter-adbe) [reply using-mobile-extensions/adobe-edge-identity/README.md:94]: Don't we have the invalid init exception?

File: using-mobile-extensions/adobe-edge-identity/README.md (hunk 1 above)
Comment (nporter-adbe) [reply using-mobile-extensions/adobe-edge-identity/README.md:117]: Yeah, the objectivec tag will provide syntax highlighting for Obj-C.

File: using-mobile-extensions/adobe-edge-identity/README.md [hunk 6]
```diff
@@ -0,0 +1,258 @@
...
+    MobileCore.registerExtensions([AEPEdgeIdentity.Identity.self, AEPIdentity.Identity.self, Edge.self], {
+    MobileCore.configureWith(appId: "yourLaunchEnvironmentID")
+  })
+  ...
+}
+```
+
+### Objective-C
+
+```text
```
Comment (nporter-adbe) [inline using-mobile-extensions/adobe-edge-identity/README.md:239]: objc tag

**Review by nporter-adbe — APPROVED** [review]
👍🏻

File: using-mobile-extensions/adobe-edge-identity/README.md (hunk 4 above)
Comment (emdobrin) [reply using-mobile-extensions/adobe-edge-identity/README.md:94]: Identity has it, but not Edge Identity.

File: using-mobile-extensions/adobe-edge-identity/README.md (hunk 1 above)
Comment (kevinlind) [reply using-mobile-extensions/adobe-edge-identity/README.md]: Updated to use the `objectivec` tag.

File: using-mobile-extensions/adobe-edge-identity/README.md (hunk 2 above)
Comment (kevinlind) [reply using-mobile-extensions/adobe-edge-identity/README.md]: Updated tag.

File: using-mobile-extensions/adobe-edge-identity/README.md (hunk 5 above)
Comment (kevinlind) [reply using-mobile-extensions/adobe-edge-identity/README.md]: I've added an FAQ page, but I'm not sure I like the FAQ format for addressing how to handle using both Identity direct and Edge Identity. I'll need to add more information in this "FAQ" page in later PRs, however, so we can change it later if needed.

**Pull request merged**